### PR TITLE
Experience

### PR DESCRIPTION
**Pull request by Oliv751**

### CODE REVIEW

File: frontend/src/pages/CandidateExperience.jsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    ) {
+      setExperiences([...experiences, newExperience]);
+    }
+  };
+
+  return (
+    <>
+      <div className="header">
+        <Header />
+      </div>
```
Comment (rocambille) [inline frontend/src/pages/CandidateExperience.jsx:37]: remove the div and move your header class inside Header

File: frontend/src/pages/CandidateExperience.jsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      <div>
+        <h1>Mes Expériences</h1>
```
Comment (rocambille) [inline frontend/src/pages/CandidateExperience.jsx:39]: div + h1 : no

File: frontend/src/pages/CandidateExperience.jsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        <form onSubmit={handleSubmit}>
+          <label>
+            Nom du poste :
+            <input
+              type="text"
+              value={title}
+              onChange={(event) => setTitle(event.target.value)}
+            />
+          </label>
+          <br />
```
Comment (rocambille) [inline frontend/src/pages/CandidateExperience.jsx:49]: no br, use css instead

File: frontend/src/pages/CandidateExperience.jsx [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+          <button type="submit">Ajouter</button>
+        </form>
+        <br />
+        <h2>Expériences professionnelles</h2>
+        {experiences.map((experience) => (
+          <div key="">
+            <h3>{experience.title}</h3>
```
Comment (rocambille) [inline frontend/src/pages/CandidateExperience.jsx:90]: div + h3 : no

File: frontend/src/pages/CandidateExperience.jsx [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+              onChange={(event) => setEndDate(event.target.value)}
+            />
+          </label>
+          <br />
+          <button type="submit">Ajouter</button>
+        </form>
+        <br />
+        <h2>Expériences professionnelles</h2>
+        {experiences.map((experience) => (
+          <div key="">
```
Comment (rocambille) [inline frontend/src/pages/CandidateExperience.jsx:89]: same key for all elements : no

**Review by rocambille — CHANGES_REQUESTED** [review]

File: frontend/src/pages/CandidateExperience.jsx (hunk 5 above)
Comment (Oliv751) [reply frontend/src/pages/CandidateExperience.jsx]: should be ok now

**Comment by Oliv751** [comment]
classname header removed

**Pull request merged**